WalletWasabi/WebClients/ChaumianCoinJoin/WasabiClient.cs

```diff
@@ -24,19 +24,15 @@ public WasabiClient(Uri baseUri, IPEndPoint torSocks5EndPoint = null) : base(bas
 
 		#region blockchain
 
-		public async Task<IEnumerable<string>> GetFiltersAsync(uint256 bestKnownBlockHash, int count)
+		public async Task<IEnumerable<FilterModel>> GetFiltersAsync(uint256 bestKnownBlockHash, Height blockHeight, int count)
```

Comment:
Can I be an ass and ask you to return string? I thought FilterModel doesn't need blockHeight. Sorry about this.

Reply:
Not a problem at all. Done.